hi @dims , is there anything wrong on the system, i have tried several times, still got failed below:

ERROR: Error fetching remote repo 'remote'
hudson.plugins.git.GitException: Failed to fetch from https://github.com/kubernetes/kubernetes
	at hudson.plugins.git.GitSCM.fetchFrom(GitSCM.java:810)
	at hudson.plugins.git.GitSCM.retrieveChanges(GitSCM.java:1066)
	at hudson.plugins.git.GitSCM.checkout(GitSCM.java:1097)
	at hudson.scm.SCM.checkout(SCM.java:485)
	at hudson.model.AbstractProject.checkout(AbstractProject.java:1269)
	at hudson.model.AbstractBuild$AbstractBuildExecution.defaultCheckout(AbstractBuild.java:607)
... skipping 2 lines ...
	at hudson.model.Run.execute(Run.java:1738)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:410)
Caused by: hudson.plugins.git.GitException: Command "/usr/bin/git -c core.askpass=true fetch --tags --progress https://github.com/kubernetes/kubernetes +refs/heads/*:refs/remotes/upstream/*" returned status code 128:
stdout:

--- a/pkg/kubelet/cm/container_manager_linux.go
+++ b/pkg/kubelet/cm/container_manager_linux.go
@@ -468,13 +468,14 @@ func (cm *containerManagerImpl) Start(node *api.Node) error {
 		return err
 	}
 	// Don't run a background thread if there are no ensureStateFuncs.
-	numEnsureStateFuncs := 0
+	hasEnsureStateFuncs := false
 	for _, cont := range cm.systemContainers {
 		if cont.ensureStateFunc != nil {
-			numEnsureStateFuncs++
+			hasEnsureStateFuncs = true
+			break
 		}
 	}
-	if numEnsureStateFuncs >= 0 {
+	if hasEnsureStateFuncs {
 		// Run ensure state functions every minute.
 		go wait.Until(func() {
 			for _, cont := range cm.systemContainers {

